pkgs/servers/code-server/default.nix

```
@@ -21,28 +21,28 @@ let
 
 in stdenv.mkDerivation rec {
   pname = "code-server";
-  version = "4.0.1";
-  commit = "7fe23daf009e5234eaa54a1ea5ff26df384c47ac";
+  version = "4.8.1";
 
   src = fetchFromGitHub {
     owner = "cdr";
```

```suggestion
    owner = "coder";
```
just noticed they renamed. Please update all other fetchers and links, too.

---

resolved